static_files/css/style.css

```diff
@@ -36,4 +63,66 @@ main{
     line-height: 22px;
 }
 
+.CathoLogo {
+
+    align-self: center;
+}
+
+
+
+.VisibleMenu {
+
+    color: #BCBF12;
```

Comment:
```suggestion
```
The below definition of color for the nav links makes this definition redundant, I suggest removing it.